src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -207,10 +230,122 @@ CAC.Control.DirectionsList = (function ($, Handlebars, UserPreferences, Utils) {
         }
     }
 
+    function shareOnFacebook(event) {
```

Comment:
These three methods probably make more sense in a separate module, since they're all share functionality and dont really appear to use too many vars in the closure.